ci-metrics/ci_listener.py

```diff
@@ -512,7 +512,7 @@ def test_ci_message(self):
                           "jenkins_job_url": "https://platform-stg-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/kernel-general-rhel-kmod/",
```

Comment:
JFYI in case we wanna make this test msg valid - this job has been move from platform-stg-jenkins to platform-jenkins.
All history builds are still there so just a quick update of hostname should be good enough.